src/Glpi/AbstractDatabase.php

```diff
@@ -1391,10 +1391,49 @@ public function getExecutionTime(): string
     /**
      * Is connected to the database
      *
-     * @retrun boolean
+     * @return boolean
      */
     public function isConnected(): bool
     {
         return $this->connected;
     }
+
+    /**
+     * Check if timezones are active.
+     *
+     * Under MySql, a command must be run to initilize data,
+     * and some privileges may be needed to read informations.
+     *
+     * @return boolean
+     */
+    public function areTimezonesActives() :bool
```

Comment:
```suggestion
    abstract public function areTimezonesActives() :bool;
```

Reply:
I do not agree. As far as I know, MySQL is an exception, no need to declare a whole method in others if it's just retuurning true.

Reply:
OK if it is an exception.